ext/pcntl/pcntl.c

```diff
@@ -654,14 +653,12 @@ PHP_FUNCTION(pcntl_signal)
 		RETURN_TRUE;
 	}
 
-	if (!zend_is_callable_ex(handle, NULL, 0, NULL, NULL, &error)) {
+	if (!zend_is_callable_ex(handle, NULL, 0, NULL, NULL, NULL)) {
 		PCNTL_G(last_error) = EINVAL;
```

Comment:
Does it actually make sense to set the last error here?

Reply:
I think so, EINVAL is for invalid arguments, and the argument is definitely not valid if this check fails.

Reply:
Well sure, but also this _used_ to return false and a warning and not a TypeError, but eh I don't think it's super important.